packages/terra-collapsible-menu-view/src/CollapsibleMenuView.jsx

```diff
@@ -5,10 +5,12 @@ import classNames from 'classnames';
 import classNamesBind from 'classnames/bind';
 import ThemeContext from 'terra-theme-context';
 import { injectIntl } from 'react-intl';
+import IconEllipses from 'terra-icon/lib/icon/IconEllipses';
```

Comment:
```suggestion
import { IconEllipses } from 'terra-icon';
```
There is no problem with functionality, but as per @sdadn initiative we are trying to move to a proper import pattern and terra-icon@3.57.0 that makes it possible was just released, would be nice to implement it that way.